org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/find-window.css.dsp

```diff
@@ -13,3 +13,24 @@
 .find-window-advanced .img-btn, .find-window-advanced .img-btn.btn-ok.z-button, .find-window-advanced .img-btn.btn-cancel.z-button {
 	margin: 0 2px;
 }
+ 
+.find-window .z-window-content{
+	padding-bottom: 0px;	
+}
+
+.find-window .status{
+    height: 20px;
+}
+
+.find-window .statusbar{
+    height: 22px;
+}
+
+.find-window .z-grid,
+.find-window .z-listbox.z-flex-item{
+	width: calc(100% - 1px) !important;
```

Comment:
you have to change the width of the .z-toolbar above the list box as well, otherwise there's a 1px difference between them.

Reply:
for consistency, it should be .z-grid.z-flex-item too.